src/domain/event/eventHero/EventHero.tsx

```diff
@@ -158,6 +159,27 @@ const EventHero: React.FC<Props> = ({ event, eventType, superEvent }) => {
                     />
                   </InfoWithIcon>
                 </Visible>
+                <Visible above="sm" className={styles.start}>
```

Comment:
about long i agree. About mobile and not mobile - i think it should be solution level and arch decision. we have to be careful with introducing device splitting. Either we use it either not, but not in single cases. Has to be well planned and defined.

Reply:
but again.. 200 lines of code. i understand what you mean. but its just because of conditional rendering here and there. if we separate to smaller components, lot of params from parent will be passed to those. and it has to be consistent.

Reply:
Dont have to do it if you dont feel like to. But it would perhaps make it more readable by splitting it to mobile and non-mobile components.

Reply:
Lets create another ticket if its reasonable. I mean this PR supposed to be a small improvement to already existing and approved PR. So the feature is not stuck in testing if its not a "must".